docs/examples/howto/add_new_algorithm.rst

```diff
@@ -0,0 +1,174 @@
+Add a new algorithm
+===================
+
+Purpose
+-------------
+
+The main part of ACTS code is located in `Core` packages. 
+For them to be used in standalone ACTS an algorithm is needed.
+Also, new ideas need to be first developed as algorithms.
+Typically the essential parts of the code are then moved to the `Core` packages 
+and the algorithm only executes it.
+
+
+
+Code Organisation
+------------------
+
+Algorithms reside in `Examples/Algorithms` are of the repository.
+The code is split into header with the algorithm class declaration 
+and a source file containing the implementation.
+
+Assuming that you want to experiment with a new seeding algorithm the files to add would be:
+`Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/MySeedingAlgorithm.h`
+and 
+`Examples/Algorithms/TrackFinding/src/MySeedingAlgorithm.h`
+
+Algorithm Class
+---------------
+The algorithm class has to inherit from {class}`ActsExamples::BareAlgorithm`
```

Comment:
The algorithm class has to inherit from {class}`ActsExamples::BareAlgorithm` and thus implement this single method: